test/Sandpit/Exscientia/Align/test_decouple.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+import BioSimSpace.Sandpit.Exscientia as BSS
+from BioSimSpace.Sandpit.Exscientia.Align._decouple import decouple
+
+
+@pytest.fixture()
+def mol():
+    # Benzene.
+    mol = BSS.Parameters.openff_unconstrained_2_0_0(
```

Comment:
Parameterising with OpenFF requires that AmberTools is installed (it requires `antechamber`). Since AmberTools is optional, please add a guard so that the test can be skipped if it's not installed. See [here](https://github.com/michellab/BioSimSpace/blob/devel/test/Process/test_amber.py) for an example. (Since it only requires `antechamber`, which must be in the `PATH`, you could instead check for that.)